core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -482,4 +485,16 @@ public boolean isAutoDownloadable() {
     public String toString() {
         return ToStringBuilder.reflectionToString(this, ToStringStyle.SHORT_PREFIX_STYLE);
     }
+
+    /**
+     *
+     * @return playback speed for this feed, or the global setting if no feed-specific setting
+     */
+    public float getFeedPlaybackSpeed() {
```

Comment:
This is a property of the feed, not a property of the feed item. I think there is only one caller. That caller can call the method on the feed directly.

Reply:
Sure thing